### PR TITLE
156 criar modulo secao

### PR DESCRIPTION
**Pull request by CarlosQuixada**

### CODE REVIEW

File: aplicacao/contest/src/main/java/ufc/quixada/npi/contest/controller/SecaoController.java [hunk 1]
```diff
@@ -8,42 +8,69 @@
...
 	}
 	
-	@RequestMapping(value="/secaoTrabalhos", method=RequestMethod.GET)
-	public String secaoTrabalhos(){
-		return "/secao/secaoTrabalhos";
-	}
-	
 	@RequestMapping(value="/cadastrarSecao")
-	public String cadastrarSecao(Secao secao){
+	public String cadastrarSecao(@RequestParam String nome_secao,@RequestParam String descricao,@RequestParam Long id_responsavel,@RequestParam Long id_evento){
```
Comment (npiuser) [inline aplicacao/contest/src/main/java/ufc/quixada/npi/contest/controller/SecaoController.java]: MINOR Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS00117)
MINOR Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS00117)
MINOR Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS00117)

File: aplicacao/contest/src/main/java/ufc/quixada/npi/contest/controller/SecaoController.java [hunk 2]
```diff
@@ -8,42 +8,69 @@
...
+		Secao secao = new Secao();
+		Pessoa responsavel = pessoaService.get(id_responsavel);
+		Evento evento = eventoService.buscarEventoPorId(id_evento);
+		
+		System.out.println(nome_secao);
```
Comment (npiuser) [inline aplicacao/contest/src/main/java/ufc/quixada/npi/contest/controller/SecaoController.java]: MAJOR Replace this use of System.out or System.err by a logger. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)

File: aplicacao/contest/src/main/java/ufc/quixada/npi/contest/controller/SecaoController.java [hunk 3]
```diff
@@ -8,42 +8,69 @@
...
+		System.out.println(descricao);
```
Comment (npiuser) [inline aplicacao/contest/src/main/java/ufc/quixada/npi/contest/controller/SecaoController.java]: MAJOR Replace this use of System.out or System.err by a logger. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)

File: aplicacao/contest/src/main/java/ufc/quixada/npi/contest/controller/SecaoController.java [hunk 4]
```diff
@@ -8,42 +8,69 @@
...
+		System.out.println(responsavel.getNome());
```
Comment (npiuser) [inline aplicacao/contest/src/main/java/ufc/quixada/npi/contest/controller/SecaoController.java]: MAJOR Replace this use of System.out or System.err by a logger. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)

File: aplicacao/contest/src/main/java/ufc/quixada/npi/contest/controller/SecaoController.java [hunk 5]
```diff
@@ -8,42 +8,69 @@
...
+		System.out.println(evento.getNome());
```
Comment (npiuser) [inline aplicacao/contest/src/main/java/ufc/quixada/npi/contest/controller/SecaoController.java]: MAJOR Replace this use of System.out or System.err by a logger. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)

**Comment by npiuser** [comment]
SonarQube analysis reported 7 issues
* ![MAJOR][MAJOR] 4 major
* ![MINOR][MINOR] 3 minor

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: aplicacao/contest/src/main/java/ufc/quixada/npi/contest/controller/SecaoController.java (hunk 1 above)
Comment (zarathon) [inline aplicacao/contest/src/main/java/ufc/quixada/npi/contest/controller/SecaoController.java]: Trocar nome das variaveis para nomeSecao, idResponsavel, idEvento

**Pull request merged**